FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.Common.UnitTests/Extensions/ConfigurationRegistrationExtensionsTests.cs

```diff
@@ -0,0 +1,96 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations.Arrow;
+using Microsoft.Health.Fhir.Synapse.Common.Exceptions;
+using Microsoft.Health.Fhir.Synapse.Common.Extensions;
+using Microsoft.Health.Fhir.Synapse.Common.Models.Jobs;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.UnitTests.Extensions
```

Comment:
seems this test is for V1? do we also validate the new config version added in the future at this file? If yes, how to distinguish different versions?

Reply:
When new version is added, a possible solution is we prepare different snapshots for history versions and runs against this test.,